packages/teams-js/package.json

```diff
@@ -17,7 +17,8 @@
     "docs:validate": "pnpm typedoc --emit none",
     "lint": "pnpm eslint ./src ./test --max-warnings 0 --fix --ext .ts",
     "prettier": "prettier --write '**/*.{ts,js,css,html}'",
-    "test": "jest"
+    "test": "jest",
+    "test:verbose": "pnpm test -- --verbose false --silent false"
```

Comment:
What is the purpose of this command?

Reply:
It's a handy (I think) command that will enable the unit tests to be run in verbose mode, which enables console logs to be seen rather than suppressed. It enables an existing logging statements to be seen when tests are run, providing an alternate way to debug issues besides stepping through them in VSCode (which also works). What are your thoughts on the usefulness of this command?

Reply:
Developers have preferences I think this command will definitely be useful, especially when writing and debugging unit test case.